package.json

```diff
@@ -28,57 +29,38 @@
     "packages/temporal-de-serializers",
     "test-packages/integration-tests",
     "test-packages/e2e-tests",
-    "test-packages/test-services",
+    "test-packages/test-services-odata-common",
+    "test-packages/test-services-odata-v2",
+    "test-packages/test-services-odata-v4",
+    "test-packages/test-services-openapi",
     "test-packages/test-services-e2e",
     "test-packages/type-tests"
   ],
   "scripts": {
-    "postinstall": "yarn compile && yarn run readme",
-    "test": "yarn workspaces run test",
-    "test:unit": "yarn lerna run test -- --ignore \"@sap-cloud-sdk/integration-tests\" --ignore \"@sap-cloud-sdk/type-tests\" --ignore \"@sap-cloud-sdk/e2e-tests\" --ignore \"@sap-cloud-sdk/nightly-tests\"",
-    "test:integration": "yarn integration-tests test",
-    "test:type": "yarn type-tests test",
-    "test:e2e": "yarn e2e-tests test",
-    "generate:test-services:v2": "yarn generate v2 && yarn lint:fix && yarn --frozen-lockfile",
-    "generate:test-services:v4": "yarn generate v4 && yarn lint:fix && yarn --frozen-lockfile",
-    "generate:common": "yarn generate common && yarn lint:fix && yarn --frozen-lockfile",
-    "generate:test-services:openapi": "yarn generate openapi && yarn lint:fix && yarn --frozen-lockfile",
-    "generate:test-services:e2e": "yarn generate e2e && yarn lint:fix && yarn --frozen-lockfile",
-    "generate:test-services": "yarn generate all && yarn lint:fix && yarn --frozen-lockfile",
-    "generate": "yarn workspace @sap-cloud-sdk/generator run compile && yarn ts-node scripts/generate-test-services.ts",
-    "compile": "yarn lerna run compile --reject-cycles",
+    "postinstall": "turbo run compile",
```

Comment:
Question: We always did compile and readme after install, why is that? Was this related to the pipeline? I would prefer to do not use any postinstall because if I want to install I just want to install.

Reply:
I tried to remove it but it breaks the pipeline right now due to the circular dependencies (which are broken up by using peer deps in the test services). Without compile, the generator (if I remember correctly) is not available during generation of test services in the pipeline.